<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_inven</class>
 <widget class="QMainWindow" name="show_inven">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Current Inventory</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="price_list">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>97</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="qty_list">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>97</y>
      <width>141</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="item_list">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>97</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="m_qty_list">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>97</y>
      <width>111</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="price_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>57</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Semilig</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price : </string>
    </property>
   </widget>
   <widget class="QLabel" name="item_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>57</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Semilig</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qty_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>57</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Semilig</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Qty :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_qty_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>57</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Semilig</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min. Qty. :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
